docs/resources/Guild_Scheduled_Event.md

```diff
@@ -101,35 +158,102 @@ Modify a guild scheduled event. Returns the modified [guild scheduled event](#DO
+| Field                 | Type                                                                                                                        | Description                                                                                |
+| --------------------- | --------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ |
+| channel_id *          | snowflake                                                                                                                   | the channel id of the scheduled event, set to `null` if changing entity type to `EXTERNAL` |
+| entity_metadata       | [entity metadata](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-entity-metadata) | the entity metadata of the scheduled event                                                 |
+| name                  | string                                                                                                                      | the name of the scheduled event                                                            |
+| privacy_level         | [privacy level](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-privacy-level)     | the privacy level of the scheduled event                                                   |
+| scheduled_start_time  | ISO8601 timestamp                                                                                                           | the time to schedule the scheduled event                                                   |
+| scheduled_end_time    | ISO8601 timestamp                                                                                                           | the time when the scheduled event is scheduled to end                                      |
+| description           | string                                                                                                                      | the description of the scheduled event                                                     |
+| entity_type *         | [event entity type](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-entity-types)  | the entity type of the scheduled event                                                     |
+| status                | [event status](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-status)             | the status of the scheduled event                                                          |
+
+\* If updating `entity_type` to `EXTERNAL`, `channel_id` [must be set to null](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-field-requirements-by-entity-type)
```

Comment:
```suggestion
\* There are some required properties if updating `entity_type` to `EXTERNAL`. `channel_id` [must be set to null](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-field-requirements-by-entity-type). A `location` must be provided in the entity_metadata. A `scheduled_end_time` is required to be set.
```

Would be nice to mention here that when changing to External `location` and `end` time is required alongside `channel_id` must be null. This is because right above this table, docs mention that all properties here are optional. P.S. My wording may not be the best so feel free to change to any wording that you prefer.

Another interesting note not sure if worth documenting though is that when you change from External to voice, `channel_id` is required but `location` is not required to be null, this will be cleared automatically by the api.